File: aussienewsapi.js
var http = require("https");

var options = {
    "method": "GET",
    "hostname": "newsapi.org",
    "port": null,
    "path": "/v2/top-headlines?country=au&apiKey=1594a48417854239bcdbd0d01fb5d86d",
    "headers": {
        "cache-control": "no-cache",
        "postman-token": "05114e36-fb51-06d7-eb5d-0d8440099e0e"
    }
};

var req = http.request(options, function (res) {
    var chunks = [];

    res.on("data", function (chunk) {
        chunks.push(chunk);
    });

    res.on("end", function () {
        var body = Buffer.concat(chunks);
        console.log(body.toString());
    });
});

req.end();

File: aussienewsapi.py
import requests

url = "https://newsapi.org/v2/top-headlines"

querystring = {"country":"us","apiKey":"1594a48417854239bcdbd0d01fb5d86d"}

headers = {
    'Cache-Control': "no-cache",
    'Postman-Token': "79a88315-e5ea-4696-9279-01fbdb54d69d"
    }

response = requests.request("GET", url, headers=headers, params=querystring)

print(response.text)